.travis.yml

```diff
@@ -64,6 +64,7 @@ env:
     - BACKENDS="Serial" NNP=ON
 
 script:
+  - export CXXFLAGS="-Wall -Wextra -pedantic -Werror"
```

Comment:
Why did you chose that over `-DCMAKE_CXX_FLAGS=...`

Reply:
No real reason..